src/text/writer.rs

```diff
@@ -470,10 +560,10 @@ mod tests {
     fn write_annotated_i64() {
         writer_test(
             |w| {
-                w.set_annotations(&["foo", "bar", "baz"]);
+                w.set_annotations(&["foo", "bar", "baz quux"]);
                 w.write_i64(7)
             },
-            "'foo'::'bar'::'baz'::7\n",
+            "foo::bar::'baz quux'::7\n",
```

Comment:
The expected output for these tests has been updated to reflect the writer's ability to omit single quotes for identifiers.